_posts/2018/07/2018-07-24-tib-hannover-fair-report.md

```diff
@@ -39,10 +38,12 @@ reuse and develop them further. Video recordings will also be available at the [
 addition, I and other participants tweeted non-stop to document the workshop for others following along remotely via the 
 hashtag [#TIBFDS](https://twitter.com/search?f=tweets&vertical=default&q=%23TIBFDS&src=typd).
 
-<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Happy to teach today and tomorrow <a href="https://twitter.com/hashtag/Python?src=hash&amp;ref_src=twsrc%5Etfw">#Python</a> at <a href="https://twitter.com/TIBHannover?ref_src=twsrc%5Etfw">@TIBHannover</a> at the <a href="https://twitter.com/hashtag/FAIR?src=hash&amp;ref_src=twsrc%5Etfw">#FAIR</a> Data and Software workshop based on <a href="https://twitter.com/thecarpentries?ref_src=twsrc%5Etfw">@thecarpentries</a>. Todays topic is <a href="https://twitter.com/hashtag/Accesible?src=hash&amp;ref_src=twsrc%5Etfw">#Accesible</a> which is now introduced by Katrin Leinweber and Angelina Kraft. <a href="https://twitter.com/hashtag/TIBFDS?src=hash&amp;ref_src=twsrc%5Etfw">#TIBFDS</a> <a href="https://t.co/6TS6j2WoVo">pic.twitter.com/6TS6j2WoVo</a></p>&mdash; Konrad Förstner (@konradfoerstner) <a href="https://twitter.com/konradfoerstner/status/1016583405612552192?ref_src=twsrc%5Etfw">July 10, 2018</a></blockquote>
-<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
-
 Katrin (left) and Angelina (pictured above) demonstrated that you can successfully pair background information on the 
 FAIR Principles with the hands-on examples taught in The Carpentries. For others hoping to better prepare for FAIR and 
 train their communities in the principles, the TIB Hannover workshop serves as an excellent starting point. I know a 
 number of us in Library Carpentry will be working with Angelina and Katrin to further develop their material.
+
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
This is duplicated. Maybe I deleting also solves the problem without moving the card / `twitter-tweet`.

Reply:
No, didn't help. https://github.com/LibraryCarpentry/librarycarpentry.github.io/commit/bd613031b0c70bf54245537b49249999c3da7d87 is the actual solution.